gqcpy/include/interfaces.hpp

```diff
@@ -699,4 +700,145 @@ void bindSimple2DMInterface(Class& py_class) {
+             "Return the orbital energies of the occupied orbitals.")
+
+        .def("orbitalEnergies",
+             &Type::orbitalEnergies,
+             "Return all of the orbital energies.")
+
+        .def("virtualOrbitalEnergies",
+             &Type::virtualOrbitalEnergies,
+             "Return the orbital energies of the virtual orbitals.");
+}
+
+
+/**
+ *  Add Python bindings for some APIs related to `complex R/U/GHF Stability matrices`.
+ * 
+ *  @tparam Class               The type of the Pybind11 `class_` (generated by the compiler).
+ *  
+ *  @param py_class             The Pybind11 `class_` that should obtain APIs related to `R/U/GHFStabilityMatrices`.
+ */
+template <typename Class>
+void bindQCModelHartreeFockComplexStabilityInterface(Class& py_class) {
+
+    // The C++ type corresponding to the Python class.
+    using Type = typename Class::type;
+    using Scalar = typename Type::Scalar;
+
+    py_class
+        .def(
+            "internal",
+            [](const Type& stability_matrices) {
+                return stability_matrices.internal();
+            },
+            "Return the internal HF stability matrix.")
+
+        .def(
+            "isInternallyStable",
+            [](const Type& stability_matrices, const double threshold) {
+                return stability_matrices.isInternallyStable(threshold);
+            },
+            py::arg("threshold") = 1.0e-05,
+            "Return a boolean, telling us if the real or complex valued internal stability matrix belongs to a stable or unstable set of parameters.")
+
+        .def(
+            "printStabilityDescription",
+            [](const Type& stability_matrices) {
+                py::scoped_ostream_redirect stream(std::cout, py::module::import("sys").attr("stdout"));
+                stability_matrices.printStabilityDescription();
+            },
+            "Print the description of the stability properties of a HF wavefunction.");
+}
+
+
+/**
+ *  Add Python bindings for some APIs related to `Real R/U/GHF Stability matrices`.
+ * 
+ *  @tparam Class               The type of the Pybind11 `class_` (generated by the compiler).
+ *  
+ *  @param py_class             The Pybind11 `class_` that should obtain APIs related to `R/U/GHFStabilityMatrices`.
+ */
+template <typename Class>
+void bindQCModelHartreeFockRealStabilityInterface(Class& py_class) {
+
+    // The C++ type corresponding to the Python class.
+    using Type = typename Class::type;
+    using Scalar = typename Type::Scalar;
+
+    py_class
+        .def(
+            "realComplex",
+            [](const Type& stability_matrices) {
+                return stability_matrices.realComplex();
+            },
+            "Return the real->complex external HF stability matrix.")
+
+        .def(
+            "isExternallyStable",
+            [](const Type& stability_matrices, const double threshold) {
+                return stability_matrices.isExternallyStable(threshold);
+            },
```

Comment:
```suggestion
            &Type::isExternallyStable,
```

Reply:
This one also raises the same error when written like that.